src/questdb/ingress.pyx

```diff
@@ -78,16 +78,22 @@ from enum import Enum
 from typing import List, Tuple, Dict, Union, Any, Optional, Callable, \
     Iterable
 import pathlib
+from cpython.bytes cimport PyBytes_FromStringAndSize
 
 import sys
 import os
 
+import numpy as np
+cimport numpy as cnp
+
+cnp.import_array()
 
 # This value is automatically updated by the `bump2version` tool.
 # If you need to update it, also update the search definition in
 # .bumpversion.cfg.
 VERSION = '2.0.4'
 
+MAX_ARRAY_DIM = 32
```

Comment:
This should probably be a `cdef`?
We might want to export this in the C symbol table: At the moment its visibility leaks into Python.

Reply:
unused variable, removed